models/issue_label.go

```diff
@@ -76,6 +78,25 @@ func (label *Label) CalOpenIssues() {
 	label.NumOpenIssues = label.NumIssues - label.NumClosedIssues
 }
 
+// CalQueryString calculates query string in issue/pulls list
+func (label *Label) CalQueryString(query []string) {
```

Comment:
You should rename `query []string` to `labels []string`.
I think that the function name is not very self-explanatory at the first place. It should be renamed into `RestoreSelectedLabels` or `LoadSelectedLabelsFromQuery`.